### PR TITLE
add infos about docker setups

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ make container-serve
 
 Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
 
+If you see errors, it probably means that the hugo container did not have enough computing resources available. To solve it, increase the amount of allowed CPU and memory usage for Docker on your machine ([MacOSX](https://docs.docker.com/docker-for-mac/#resources) and [Windows](https://docs.docker.com/docker-for-windows/#resources)).
```
Comment (kbhawkey) [inline README.md:48]: Hi @remyleone .

Does it make sense to add this note immediately after the make "container" instructions?

Comment (remyleone) [reply README.md:46]: It does :) I've fixed it.

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by Sea-n** [comment]
Hey @remyleone,

Seems you have open the branch in the wrong repo.

Could you delete this `remyleone-patch-1` branch from k/website? Thanks.

**Pull request merged**